medicines/api/infrastructure/non-prod/deployment.yml

```diff
@@ -15,14 +15,21 @@ spec:
     metadata:
       labels:
         app: api
+        env: non-prod
     spec:
       containers:
-        - image: api
+        - image: mhraproductsnonprod.azurecr.io/products/api
           name: api
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 8000
```

Comment:
if this port is the same as the main port, you can omit it

Reply:
This field is required